gpytorch/variational/lmc_variational_strategy.py

```diff
@@ -174,18 +179,15 @@ def __call__(self, x, task_indices=None, prior=False, **kwargs):
             :class:`~gpytorch.distributions.MultivariateNormal`.
 
         :param x: Input locations to evaluate variational strategy
-        :type x: torch.Tensor (... x N x D)
```

Comment:
It's a bit unfortunate to remove the shape annotation here in favor of the type annotations. Can we keep it around for now? Hopefully down the road we can use shape annotations directly in the types as well (e.g. as in the works in https://github.com/cornellius-gp/linear_operator/pull/42).